charts/opensearch/values.yaml

```diff
@@ -196,10 +196,17 @@ extraContainers: []
   #   image: busybox
   #   command: ['do', 'something']
 
-extraInitContainers: []
-  # - name: do-somethings
-  #   image: busybox
-  #   command: ['do', 'something']
+extraInitContainers:
+ - name: volume-chmod
+   image: busybox
+   command: ['sh', '-c']
+   args:
+     - 'chown -R 1000:1000 /usr/share/opensearch'
+   securityContext:
+     runAsUser: 0
+   volumeMounts:
+     - mountPath: /usr/share/opensearch/data
+       name: opensearch-cluster-master
```

Comment:
If this is necessary, then it should be default to the statefulset.

Reply:
done !